http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/LabelComponent.scala

```diff
@@ -16,19 +16,18 @@ class LabelTable(tag: Tag) extends Table[LabelRecord](tag, "LABEL") {
   def key = column[String]("key", O.Length(254))
   def value = column[String]("value", O.Length(254))
 
-  def uniqueKey = index("IDX_LABEL_UNIQUE", (resourceId, resourceType, key), unique = true)
+  def pk = primaryKey("label_pk", (resourceId, resourceType, key))
```

Comment:
Maybe specify the constraintName in liquibase so the name here matches. Seems you can do:
```
<addPrimaryKey tableName="LABEL" columnNames="resourceId, resourceType, key" contstraintName="PK_LABEL" />
```